serverless/service_mesh/serverless-ossm-tls-with-kourier.adoc

```diff
@@ -1,31 +1,31 @@
 include::modules/serverless-document-attributes.adoc[]
 include::modules/ossm-document-attributes.adoc[]
-[id="serverless-ossm-tls"]
-= Configuring Transport Layer Security for a custom domain using {ProductName}
+[id="serverless-ossm-tls-with-kourier"]
+= Configuring Transport Layer Security for a custom domain using {ProductName} and Kourier
 :context: serverless-ossm-tls
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 You can create a Transport Layer Security (TLS) key and certificates for a custom domain and subdomain using {ProductName}.
 
-[NOTE]
-====
-{ServerlessProductName} is compatible only with full implementations of either {ProductName} 1.x or 2.x. {ServerlessProductName} does not support custom usage of some 1.x resources and some 2.x resources in the same deployment. For example, upgrading to 2.x while still using the control plane `maistra.io/v1` spec is not supported.
-====
-
```

Comment:
It wasn't removed, it was just moved in this page (look below) - for OCP docs we're supposed to avoid having a bunch of notes right next to each other, so I moved it to where we talk about installing SM as a prereq since it makes more sense there anyway